wpilibc/src/main/native/include/frc/smartdashboard/SendableChooser.h

```diff
@@ -113,17 +107,47 @@ class SendableChooser : public SendableChooserBase {
     }
   }
 
+  /**
+   * Returns the name of the selected option.
+   *
+   * If there is none selected, it will return the default option's name. If
+   * there is none selected and no default, it will return an empty string.
+   *
+   * @return The name of the option selected
+   */
+  std::string_view GetSelectedName() const {
```

Comment:
```suggestion
  std::string GetSelectedName() const {
```

It's not safe to return a string_view here.  It needs to return a copy (a std::string) because the access is mutex-protected, so (theoretically) another thread could come in and modify m_selected before the string_view is accessed.